runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -552,6 +553,15 @@ public void show()
 			requestFocus();
 			giveClientFocus();
 			log.info("Showing frame {}", frame);
+
+			// Close sidebar if config is set for it.
+			if (config.rememberSidebarState())
```

Comment:
It probably isn't necessary to have a config option for this

Reply:
Sorry for the double push, I only saw the first comment. But this should also be solved now.